src/main/java/teammates/storage/entity/AdminEmail.java

```diff
@@ -98,7 +88,7 @@ public void setSendDate(Date sendDate) {
     }
 
     public String getEmailId() {
-        return this.emailId;
+        return Key.create(AdminEmail.class, this.emailId).toWebSafeString();
```

Comment:
```patch
-    @PrimaryKey
-    @Persistent(valueStrategy = IdGeneratorStrategy.IDENTITY)
-    @Extension(vendorName = "datanucleus", key = "gae.encoded-pk", value = "true")
-    private String emailId;
```
The flag `gae.encoded-pk` made JDO populate the value with the websafe string of the key, and `getEmailId` would return that. Changing the schema is beyond the scope of this PR.